monitor/docker-compose.yml

```
@@ -36,4 +36,11 @@ services:
       - GF_SECURITY_ADMIN_PASSWORD=camunda
       - GF_USERS_ALLOW_SIGN_UP=false
 
+  zeebe:
+    image: camunda/zeebe:current-test
```

```suggestion
    image: camunda/zeebe:SNAPSHOT
```

It's more portable this way in general

---

If you want an easy override, we do this in other places:

```
image: camunda/zeebe:${ZEEBE_VERSION:-SNAPSHOT}
```